src/app/components/Basic/index.js

```diff
@@ -1,4 +0,0 @@
-export {
```

Comment:
^^
should be on `src/app/commands.js -> src/app/commands.ts`

I'd leave `commands.js` as is (it will stay a `.js` for some more time even on alinity pro). Even as a `.ts`, you don't get many benefits as of now due to missing typings for avenger etc.